changelog/fragments/example1.yaml

```diff
@@ -0,0 +1,22 @@
+entries:
```

Comment:
Will these examples be erroneously added to the generated changelog when `make gen-changelog` is run?

Reply:
I'll remove those before merging. Just wanted those in there so I and others could experiment with it.